bin/refresh-styles

```diff
@@ -1,16 +1,12 @@
 #! /usr/bin/env bash
 
-# DOC: Regenerate tailwind stylesheets.
+# DOC: Run npm in the dev environment's docker container.
```

Comment:
I love the changes here! Were these changes generated by some automated find/replace script you had? If so, would you mind describing how this was performed in the PR description (would make for an easier review knowing how the changes were generated).

If this was a script, a way to easily include it for review would be to:
1. Push a commit with the script that was used added to the repo
2. Push a separate commit that deletes the script
3. Reference the specific commit / file in a link from the PR description.

Reply:
Oh yeah thats a good idea on how to include the automated script I used temporarily

But yeah for this one I shouldn't have replaced the comment. I basically copied the `bin/npm` file into `bin/refresh-styles` and just changed the line with the command

Reply:
I have that file in a separate repo. Maybe I can commit it. I basically took the functions of the old tailwind.config.js and used those

Reply:
I think I've got the styles mostly working, but theres still some issues. E.g. when you try to publish a problem the button that pops up to comfirm doesn't show up, even though the BG kinda goes greyish